DataFormats/Histograms/interface/DQMToken.h

```diff
@@ -25,5 +25,12 @@ class DQMToken
       DQMToken() {}
 };
 
+class DQMRunToken
```

Comment:
There really is not a need for a seperate type just for Run.

Reply:
@Dr15Jones I thought so. But having two `produce` calls with the same Tokens gave me an error, and this was the quick fix.